deployment/OMS.md

```diff
@@ -0,0 +1,37 @@
+## What is log analytics?
```

Comment:
Could you change the name of the file to match the title of the section? i.e. LogAnalytics.md

Also, please change casing of Log Analytics to be consistent throughout the doc.